task/deprecated-image-check/0.3/MIGRATION.md

```diff
@@ -1,10 +1,28 @@
 # Migration from 0.2 to 0.3
 
-Workspace used by the `deprecated-image-check` is removed. This is not required as it doesn't need any PVCs. 
+Workspace used by the `deprecated-image-check` is removed. This is not required as it doesn't need any PVCs.
 
 ## Action from users
 
 Update files in Pull-Request created by RHTAP bot:
 - Search for the task named `deprecated-base-image-check`
-- Remove the workspaces section from [deprecated-image-check.yaml](https://github.com/redhat-appstudio/build-definitions/blob/main/task/deprecated-image-check/0.2/deprecated-image-check.yaml)
-- Replace `$(workspaces.test-ws.path)` with `/tekton/home`
+- Remove the workspaces section from the task
+- Do NOT remove workspaces from the other tasks
+
+Example how the section should look like:
+```yaml
+  - name: deprecated-base-image-check
+    # ...
+    taskRef:
+       params:
+       - name: name
+         value: deprecated-image-check
+     #
+     # more attributes here
+     # ...
+     #
+     # remove section below
+     workspaces:
+     - name: test-ws
+       workspace: workspace
+```
```

Comment:
```suggestion
Example how the section should look like before and after the change:

BEFORE:
```yaml
  - name: deprecated-base-image-check
      params:
      - name: BASE_IMAGES_DIGESTS
        value: $(tasks.build-container.results.BASE_IMAGES_DIGESTS)
      taskRef:
        params:
        ...
      when:
      - input: $(params.skip-checks)
        operator: in
        values:
        - "false"
      workspaces:            <--- remove this
      - name: test-ws        <--- remove this 
        workspace: workspace <--- remove this
\```

AFTER:
```yaml
  - name: deprecated-base-image-check
      params:
      - name: BASE_IMAGES_DIGESTS
        value: $(tasks.build-container.results.BASE_IMAGES_DIGESTS)
      taskRef:
        params:
        ...
      when:
      - input: $(params.skip-checks)
        operator: in
        values:
        - "false"
```
```

![image](https://github.com/redhat-appstudio/build-definitions/assets/43273420/f017c3cd-a0f7-41e6-863d-a81f1de89575)

Reply:
But maybe its too long for a migration note?

Reply:
updated